ddp-workspace/projects/ddp-dsm-ui/src/app/sharedLearningUpload/components/uploadFile/uploadFile.component.ts

```diff
@@ -112,7 +119,7 @@ export class UploadFileComponent implements OnDestroy {
   }
 
   public get shouldDisableButton(): boolean {
-    return !this.isFileSelected || this.uploadStatus === HttpRequestStatusEnum.SUCCESS;
+    return !this.isFileSelected || this.uploadStatus === HttpRequestStatusEnum.SUCCESS || noConnectionToBucket;
```

Comment:
What is `noConnectionToBucket`?

Reply:
Apologies, I should have marked this as draft--still working on this part.